keyboards/electronlab/klor/config.h

```diff
@@ -0,0 +1,148 @@
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+// ┌─────────────────────────────────────────────────┐
+// │ p i n   c o n f i g                             │
+// └─────────────────────────────────────────────────┘ 
+
+#define I2C1_SCL_PIN GP3
+#define I2C1_SDA_PIN GP2
+
+// ┌─────────────────────────────────────────────────┐
+// │ k e y   m a t r i x                             │
+// └─────────────────────────────────────────────────┘ 
+
+#define MATRIX_ROWS 8
+#define MATRIX_COLS 6
+#define DEBOUNCE 5
+
+
+// ┌─────────────────────────────────────────────────┐
+// │ s p l i t   c o m m u n i c a t i o n           │
+// └─────────────────────────────────────────────────┘ 
+
+#define USE_SERIAL
+#define SERIAL_USE_MULTI_TRANSACTION
+#define SPLIT_USB_DETECT
+
+//Half Duplex communication
+#define SERIAL_USART_TX_PIN GP1     // USART TX pin
+
+//Full Duplex communication
+//#define SERIAL_USART_TX_PIN GP4     // USART TX pin
+//#define SERIAL_USART_RX_PIN GP1     // USART RX pin
+//#define SERIAL_USART_FULL_DUPLEX
+//#define SERIAL_USART_PIN_SWAP
+
+
+// ┌─────────────────────────────────────────────────┐
+// │ e n c o d e r s                                 │
+// └─────────────────────────────────────────────────┘ 
+
+#define ENCODER_RESOLUTION { 2 }
+#define ENCODER_RESOLUTIONS_RIGHT { 2 }
+
+// ┌─────────────────────────────────────────────────┐
+// │ o l e d s                                       │
+// └─────────────────────────────────────────────────┘
+
+#ifdef OLED_ENABLE
+#    define OLED_DISPLAY_128X64
+#    define SPLIT_OLED_ENABLE
+#    define OLED_FONT_H  "./glcdfont.c"
+#endif
+
+
+// ┌─────────────────────────────────────────────────┐
+// │ r g b   l e d s                                 │
+// └─────────────────────────────────────────────────┘ 
+
+#define RGBLED_NUM 42
+#define RGBLED_SPLIT { 21, 21 }
+#define RGB_MATRIX_LED_COUNT RGBLED_NUM
+
+#ifdef RGB_MATRIX_ENABLE
+#    define RGB_MATRIX_STARTUP_MODE RGB_MATRIX_BREATHING
+#    define SPLIT_TRANSPORT_MIRROR true
+#    define DRIVER_LED_TOTAL 42 // Number of LEDs
+#    define RGB_MATRIX_SPLIT { 21, 21 }
+#    define RGB_MATRIX_TIMEOUT 120000
+#    define RGB_MATRIX_SLEEP
+#    define RGB_MATRIX_LED_FLUSH_LIMIT 16
+#    define RGB_MATRIX_MAXIMUM_BRIGHTNESS 255
+#    define RGB_MATRIX_STARTUP_HUE 299
+#    define RGB_MATRIX_DEFAULT_ON true
+#    define RGB_MATRIX_DEFAULT_SAT 255
+#    define RGB_MATRIX_DEFAULT_SPD 127
+#    define RGB_MATRIX_STARTUP_VAL RGB_MATRIX_MAXIMUM_BRIGHTNESS
+#    define RGB_DISABLE_WHEN_USB_SUSPENDED
+#    define RGB_MATRIX_KEYPRESSES
+#endif 
+
+
```

Comment:
```suggestion
```
All this should be configured in JSON (although you can't express the conditional logic to enable `SPLIT_TRANSPORT_MIRROR` there).  Note that some of these names, like `RGB_DISABLE_WHEN_USB_SUSPENDED`, are obsolete.

Reply:
Agreed. Change is done and committed.